From 42c09a78d9ed2dc393e76308f4fce000535674e6 Mon Sep 17 00:00:00 2001
From: Lidong Zhong <lzhong@suse.com>
Date: Mon, 9 Mar 2015 02:29:36 +0000
Subject: [PATCH] objdb: Fix incorrect using lock

It obviously is a typo, but it dose cause coredump easily

Signed-off-by: Lidong Zhong <lzhong@suse.com>
Reviewed-by: Jan Friesse <jfriesse@redhat.com>
---
 exec/objdb.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/exec/objdb.c b/exec/objdb.c
index 5d7c124..da1270d 100644
--- a/exec/objdb.c
+++ b/exec/objdb.c
@@ -1412,7 +1412,7 @@ static int object_priv_get (
 	int res;
 	struct object_instance *object_instance;
 
-	objdb_unlock();
+	objdb_lock();
 	res = hdb_handle_get (&object_instance_database,
 		object_handle, (void *)&object_instance);
 	if (res != 0) {
-- 
1.7.1

